layers/meta-tegrademo/scripts/setup-udev-rules

```diff
@@ -0,0 +1,38 @@
+#!/bin/sh
+if [ "$(id -u)" -ne 0 ]; then
+    echo "You may need to run this script as sudo if you see permission errors"
+    user=`whoami`
```

Comment:
Maybe you should exit here, right ?

Reply:
I don't have a strong preference between warning that you might fail and exiting.  I suspect it's possible you could succeed if you had the right permissions setup with a non root user account.